modules/detections_and_time.py

```diff
@@ -114,13 +104,14 @@ def __init__(self, class_private_create_key, timestamp: float):
         self.detections = []
         self.timestamp = timestamp
 
-    def __repr__(self) -> str:
-        representation = \
-            str(self.__class__) \
-            + ", time: " + str(int(self.timestamp)) \
-            + ", size: " + str(len(self))
-
-        representation += "\n" + repr(self.detections)
+    def __str__(self) -> str:
+        """
+        To string.
+        """
+        representation = (
+            f"{self.__class__}, time: {int(self.timestamp)}, size: {len(self)}\n"
+            + f"{self.detections}"
+        )
         return representation
```

Comment:
Keep __str__() function consistent with other modules. Either just return the string or always assign representation = f"...." and return representation